docs/UserGuide.md

```diff
@@ -91,9 +91,9 @@ LE TRACKER is a gamified tracking application that allows fast typist to easily
 
 ### Delete
 
-- `delete-module {module_code}`: Deletes a module from Le Tracker
-- `delete-lecture /module {module_code} /lecture {lecture_id}`: Deletes the specified lecture from the specified module
-- `delete-video /module {module_code} /lecture {lecture_id} /video {video_id}`: Deletes the specified video from the specified lecture from the specified module
+- `delete {module_code}`: Deletes module(s) from Le Tracker
+- `delete {lecture_name} /mod {module_code}`: Deletes the specified lecture(s) from the specified module
+- `delete {video_name} /mod {module_code} /lec {lecture_name}`: Deletes the specified video(s) from the specified lecture from the specified module
```

Comment:
`/mod` and `/lec` are optional arguments and should be enclosed in square brackets
To specify that an argument can take multiple inputs, refer to `tag`